File: WebApi/ClientApp/src/app/models/testung.ts
export class Testung
{
    id?: number;
    name: string;
    date: Date | string;
    chapters: TestungChapter[];
    patientId: number | null;
}

export interface TestungQuestion
{
    id: number | null;
    type: string;
    value: string;
    label: string;
    chapterId: number | null;
}


export interface TestungChapter
{
    id: number | null;
    name: string;
    score: number | null;
    questions: TestungQuestion[];
    testungId: number | null;

}

File: rl-contract/Models/Documents/Testung.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace rl_contract.Models
{
    public class Testung
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int? Id { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }

        public List<TestungChapter> Chapters { get; set; }

        public int? PatientId { get; set; }
        /*public Patient Patient { get; set; }*/

    }
}
